### DISCUSSION TITLE
Quiz response data

### DISCUSSION THREAD

**Discussion by sotoblanco** [freeCodeCamp/Developer_Quiz_Site#648] [Ideas]
Do you collect data about user responses to the quiz? If we have that, we could identify misleading or unfair questions by evaluating their distribution. Those that do not fit with the expected distribution we could fix, modify or delete, this can help to improve the learner experience.

**Comment by jdwilkin4** [comment]
No, we don't save user responses after they finish the game. 
If there is a question that you find misleading, then it would be best to open an issue first. 👍

**Reply by sotoblanco** [reply]
Ok. Yes. That make sense. Thank you!!

**Reply by ProfJoanna** [reply]
I noticed a few mistakes in some questions, and agree with @sotoblanco
